app/src/main/java/com/yacgroup/yacguide/DescriptionActivity.kt

```diff
@@ -126,6 +126,14 @@ class DescriptionActivity : TableActivity() {
         }
     }
 
+    override fun showParent(v: View) {
+        super.showParent(v)
+
+        val intent = Intent(this, RouteActivity::class.java)
+        intent.putExtra(IntentConstants.ROCK_KEY, _route?.parentId)
```

Comment:
A description without route should be impossible. So please assert here (using `!!` instead of `?`)

Reply:
An assert leads to an immediate crash here. I would prefer if the app would continue working, if such cases ever occur. We could however skip the entire block via _route.let{}

Reply:
Ok